packages/node-core/src/db/sync-helper.ts

```diff
@@ -561,13 +584,58 @@ export const createIndexQuery = (indexOptions: IndexesOptions, tableName: string
     `CREATE ${indexOptions.unique ? 'UNIQUE ' : ''}INDEX IF NOT EXISTS "${indexOptions.name}" ` +
     `ON "${schema}"."${tableName}" ` +
     `${indexOptions.using ? `USING ${indexOptions.using} ` : ''}` +
-    `(${indexOptions.fields.join(', ')})`
+    `(${indexOptions.fields.join(', ')});`
   );
 };
 
 // columns
 export const dropColumnQuery = (schema: string, columnName: string, tableName: string): string =>
   `ALTER TABLE  "${schema}"."${modelToTableName(tableName)}" DROP COLUMN IF EXISTS ${columnName};`;
 
-export const createColumnQuery = (schema: string, tableName: string, columnName: string, attributes: string): string =>
-  `ALTER TABLE IF EXISTS "${schema}"."${tableName}" ADD COLUMN IF NOT EXISTS "${columnName}" ${attributes};`;
+export const createColumnQuery = (schema: string, table: string, columnName: string, attributes: string): string =>
+  `ALTER TABLE IF EXISTS ${escapedName(schema, table)} ADD COLUMN IF NOT EXISTS "${columnName}" ${attributes};`;
```

Comment:
@bz888 I think this should probably be, this should fail if the table doesn't exist
```suggestion
  `ALTER TABLE ${escapedName(schema, table)} ADD COLUMN IF NOT EXISTS "${columnName}" ${attributes};`;
```